### PR TITLE
Replace announcement "text" field break tags with newlines.

### PR DESCRIPTION
**Pull request by montehurd**
https://phabricator.wikimedia.org/T166356

( Note: bearND was able to fix this upstream for this announcement, so hopefully the survey delivered to app store version will be fixed by the time you read this. See the phab ticket for details. )

# Before

screen shot 2017-05-25 at 5 26 02 pm

# After

screen shot 2017-05-25 at 6 58 17 pm

### CODE REVIEW

File: WMF Framework/WMFAnnouncement.m [hunk 1]
```diff
@@ -46,6 +46,12 @@ + (NSValueTransformer *)imageURLJSONTransformer {
         }];
 }
 
++ (MTLValueTransformer *)textJSONTransformer {
+    return [MTLValueTransformer transformerUsingForwardBlock:^id(NSString *text, BOOL *success, NSError *__autoreleasing *error) {
+        return [text stringByReplacingOccurrencesOfString:@"<br>" withString:@"\n"];
```
Comment (amrox) [inline WMF Framework/WMFAnnouncement.m]: is it possible there could be a `<br/>` or `<br />`?

Comment (montehurd) [inline WMF Framework/WMFAnnouncement.m]: It's a human-edited field so that's possible though unlikely. I think @joewalsh's comment is on-point too...

**Comment by montehurd** [comment]
Good points. I'll look into adding an upstream test.

**Comment by montehurd** [comment]
Closing in favor of adding upstream test.

**Pull request closed without merging**